docs/openstack-servers.md

```diff
@@ -0,0 +1,150 @@
+# Openstack Servers
+
+To read more about Openstack Servers using the [upstream docs](https://docs.openstack.org/python-openstackclient/pike/cli/command-objects/server.html).
```

Comment:
lets put the latest docs link here:
https://docs.openstack.org/python-openstackclient/latest/cli/command-list.html

Reply:
This has been updated.